webhooks/2025-04-15.yml

```diff
@@ -2551,172 +2563,111 @@ components:
         - is_recurring
-        - is_prorated
         - amount
-        - period
-        - interval
-        - has_base_fee
         - currency
-        - annual_monthly_amount
-        - publicly_visible
     BillingSubscriptionItem:
       type: object
       properties:
         id:
           type: string
           description: Unique identifier for the subscription item
-        status:
-          type: string
-          description: Current status of the subscription item (e.g., "active", "canceled", "past_due", "incomplete")
-        credit:
-          $ref: '#/components/schemas/BillingSubscriptionCredit'
-          description: Available credit information for the subscription item
-        proration_date:
+        object:
           type: string
-          description: Date used for proration calculations when subscription changes occur
-        plan_period:
+          description: String representing the object's type
+          enum:
+            - commerce_subscription_item
+        status:
           type: string
-          description: Billing period for the subscription item (e.g., "month", "annual")
-        period_start:
+          description: Current status of the subscription item
+          enum:
+            - abandoned
+            - active
+            - canceled
+            - ended
+            - expired
+            - incomplete
+            - past_due
+            - upcoming
+        created_at:
           type: integer
           format: int64
-          nullable: true
-          description: Unix timestamp in milliseconds when the subscription item period started. Null for non-recurring items.
-        period_end:
+          description: Unix timestamp in milliseconds when the subscription item was created
+        updated_at:
           type: integer
           format: int64
-          nullable: true
-          description: Unix timestamp in milliseconds when the subscription item period ends. Null for non-recurring items.
-        canceled_at:
+          description: Unix timestamp in milliseconds when the subscription item was most recently updated
+        period_start:
           type: integer
           format: int64
-          nullable: true
-          description: Unix timestamp in milliseconds when the subscription item was canceled. Null if not canceled.
-        past_due_at:
+          description: Unix timestamp in milliseconds when the subscription item period started
+        period_end:
           type: integer
           format: int64
+          description: Unix timestamp in milliseconds when the subscription item period ended
+        interval:
+          type: string
+          description: How frequently the payer will be charged while this subscription item is active
+          enum:
+            - month
+            - annual
+        plan_id:
+          type: string
+          description: Unique identifier for the plan associated with this subscription item
           nullable: true
```

Comment:
When would a `BillingSubscriptionItem` not have a valid `plan_id` attached to it?